pkg/kubectl/cmd/config/create_authinfo.go

```diff
@@ -296,3 +323,35 @@ func (o createAuthInfoOptions) validate() error {
 
 	return nil
 }
+
+func (o *createAuthInfoOptions) checkPrompts() (err error) {
+	var result string
+
+	prompter := prompt.NewPrompter(bufio.NewReader(os.Stdin))
```

Comment:
Don't use `os.Stdin` directly since prompting can mess up with unit tests. Pass the Reader to `NewCmdConfigSetAuthInfo` and store it in the `createAuthInfoOptions`. Some other commands also take the Reader in `NewCmd*`.